src/test/java/com/rtb/projectmanagementtool/project/servlets/DisplayProjectsServletTest.java

```diff
@@ -0,0 +1,108 @@
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.dev.LocalDatastoreService;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.gson.Gson;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import javax.servlet.http.*;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.*;
+
+public class DisplayProjectsServletTest extends Mockito {
+  private HttpServletRequest request;
+  private HttpServletResponse response;
+  private DatastoreService datastore;
+  private ProjectController projectController;
+  private DisplayProjectsServlet servlet;
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig()
+              .setAutoIdAllocationPolicy(LocalDatastoreService.AutoIdAllocationPolicy.SEQUENTIAL));
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+    request = mock(HttpServletRequest.class);
+    response = mock(HttpServletResponse.class);
+    datastore = DatastoreServiceFactory.getDatastoreService();
+    projectController = new ProjectController(datastore);
+    servlet = new DisplayProjectsServlet();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void displayProjectsWithUser() throws Exception {
+    StringWriter stringWriter = new StringWriter();
+    PrintWriter printWriter = new PrintWriter(stringWriter);
+    when(response.getWriter()).thenReturn(printWriter);
+    when(request.getParameter("query")).thenReturn("isIn");
+
+    // Create project - user should be in this one
+    ProjectData newProject = new ProjectData("Project1", "Desc", 2l);
```

Comment:
If we're going to be adding multiple projects at a time, maybe look into expanding your controller to allow for adding a list of projects (to consolidate the datastore call into a single insert)